ignite/metrics/running_average.py

```diff
@@ -9,8 +9,10 @@ class RunningAverage(Metric):
         src (Metric or None): input source: an instance of :class:`ignite.metrics.Metric` or None. The latter
             corresponds to `engine.state.output` which holds the output of process function.
         alpha (float, optional): running average decay factor, default 0.98
-        output_transform (Callable, optional): a function to use to transform the output if `src` is None and
-            corresponds the output of process function. Otherwise it should be None.
+        output_transform (callable, optional): a callable that is used to transform the
```

Comment:
What ???

Reply:
@anmolsjoshi in case of `RunningAverage`, the role of `output_transform` is the same as for `Metric` and in addition there is a condition on when it should be defined and when it should be None depending on `src`.